### ISSUE TITLE
[Screen reader - Bot Framework Web Chat - AI Chatbot]: Screen reader user is unable to access the chat bot responses.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5526]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader user is not able to access the chat bot responses. both in forms mode and browse mode. It is confusing for the screen reader user to understand the responses.

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://azure.microsoft.com/en-us/?at_preview_token=4FhxMZZq5RXQNNpOOuFqCKRlAHybD84mN9iFqfn8BXs&at_preview_index=1_2&at_preview_listed_activities_only=true

### How to reproduce the issue?

1. Go to the [Cloud Computing Services | Microsoft Azure](https://azure.microsoft.com/en-us/?at_preview_token=4FhxMZZq5RXQNNpOOuFqCKRlAHybD84mN9iFqfn8BXs&at_preview_index=1_2&at_preview_listed_activities_only=true)
2. turn on NVDA.
3. go to the edit box of "AI Powered Assistant."
4. Type a question. 
5. Submit the question.
6. Follow the keyboard instructions and press Shift + tab to go the chat history.
7. Observe the NVDA behavior.

### What do you expect?

After the user submits a question screen reader user should announce the response. 

When pressed shift + tab to go to chat history, pressing down and up arrow should make the screen reader announce the response not just interactive elements in the response.

When in forms mode, screen reader should announce the response not just document 35 of 35

When in browse mode, there should be no keyboard trap when using tab key for the interactive elements.

Also, there should be a heading at the beginning of each new question/answer pair to allow a user to move to the beginning of the question response in browse mode.

### What actually happened?

By default, NVDA is in “forms” mode. In this mode, when a user follows the instructions provided to press Shift + tab to go to the chat history and press up and down arrow to read the message, the screen reader announces the card, links and attachments. The screen reader does not announce the content of the response.

When changed to scan mode from forms mode to read the message, focus jumps to the bottom of the message and user has to keep pressing up arrow to go to the top to read the message from beginning.

If, on the other hand, the user tries to switch to “browse” mode before following the instructions, there is a keyboard trap when pressing shift+tab causes focus to keep rotating between the links. User has no way out to move out of the trap.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/b18898f5-62eb-4c94-b913-fb60f9d76618

### MAS reference

https://aka.ms/MAS1.3.1

### Adaptive Card JSON

```json

```

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#HCL;#Accessibility;#AILimited;#Benchmark;#BotFrameworkWebChat-Oct24;#MAS1.3.1;#Win11-Edge(Chromium);
